### DISCUSSION TITLE
Comment with your upstream services

### DISCUSSION THREAD

**Discussion by jc21** [NginxProxyManager/nginx-proxy-manager#1203] [General]
In order to provide support for the most common upstream (proxied) services, it's helpful to know what they are. Comment with your services and I'll update this main description with them:

- Plex Media Server
- Sonarr
- Radarr
- Couchpotato
- Sickbeard
- Confluence
- Jira
- Transmission Web
- Home Assistant
- Lidarr
- Verdaccio
- Portainer
- Router Admin
- Bitbucket Server
- NzbGet
- SabNZB
- Emby (https://emby.media) / Jellyfin (https://github.com/jellyfin/jellyfin)
- Ombi (https://ombi.io/)
- Deluge (https://deluge-torrent.org)
- NZB Hydra (https://github.com/theotherp/nzbhydra2)
- FreePBX (https://www.freepbx.org/)
- Bitwarden (https://bitwarden.com/) / Bitwarden_RS (https://github.com/dani-garcia/bitwarden_rs)
- osTicket (https://osticket.com/)
- Gitea (https://gitea.io)
- NextCloud (https://nextcloud.com) / Owncloud (https://owncloud.org)
- Matrix (https://matrix.org)
- Bitwarden
- Unifi controller (https://www.ui.com/software/)
- Jitsi (https://jitsi.org/)

**Comment by Cerothen** [comment]
Emby (https://emby.media) / Jellyfin (https://github.com/jellyfin/jellyfin)
Ombi (https://ombi.io/)
Deluge (https://deluge-torrent.org)
NZB Hydra (https://github.com/theotherp/nzbhydra2)
FreePBX (https://www.freepbx.org/)
Bitwarden (https://bitwarden.com/) / Bitwarden_RS (https://github.com/dani-garcia/bitwarden_rs)
osTicket (https://osticket.com/)
Gitea (https://gitea.io)
NextCloud (https://nextcloud.com) / Owncloud (https://owncloud.org)
Matrix (https://matrix.org)

**Comment by WesSec** [comment]
Plex Media Server
Sonarr
Radarr
Home Assistant
Verdaccio
Portainer
NzbGet
Ombi
Deluge
NZB Hydra
Bitwarden
NextCloud (https://nextcloud.com) / Owncloud (https://owncloud.org)

**Comment by Jason-nz** [comment]
Unifi controller (https://www.ui.com/software/) needs X-Frame-Options changed when proxying to allow embedding in iframes etc.

**Comment by OhHeyAlan** [comment]
Currently running 
Docker
Mayan-EDMS (https://www.mayan-edms.com/)
Portainer 
Watchtower (https://hub.docker.com/r/v2tec/watchtower/)
Gitlab CE
Gitlab Runner
Private Registry
CFSSL (cfssl.org)

Everything is ran as a Docker container.

**Comment by beloniki** [comment]
Running on docker:
Redmine
Rocket-chat
NextCloud
PhpMyAdmin
PhpLdapAdmin
Password Self Service (https://github.com/fjudith/docker-pwm)
Wordpress
Portainer
ELK
Zabbix
AWX

System service:
Gitlab ce
Confluence

**Reply by krimlon** [reply]
do you have any configuration suggestions for rocket-chat?

**Comment by Unthred** [comment]
airsonic
chronograf
funkwhale
glances
goaccess
grafana
jackett
monitorr
nessus
now showing
qbittorrent
telegraf
tautulli
Plex Media Server
Sonarr
Radarr
Transmission Web
Home Assistant
Lidarr
NzbGet
SabNZB
Ombi (https://ombi.io/)
NZB Hydra (https://github.com/theotherp/nzbhydra2)
NextCloud (https://nextcloud.com) / Owncloud (https://owncloud.org)
Unifi controller (https://www.ui.com/software/)

**Comment by sascha-hemi** [comment]
Jitsi (https://jitsi.org/)

**Comment by Indemnity83** [comment]
Unraid

I'm actually surprised this wasn't already on this list. I realize it doesn't really "need" a reverse proxy since its typically the one service on Port 80 for many users but Lets Encrypt and a common domain mean it isn't useless. 

As an interesting side note, some things don't work right in Unraid unless you add a `gzip off;` directive to custom config. The gzip buffering will cause pages that basically echo out status updates to appear to "hang"

**Comment by schneekluth** [comment]
[Shaarli](https://github.com/shaarli/Shaarli)

**Comment by au190** [comment]
Hi is the https://emby.media - emby revers proxy working for somebody ? I get Forbidden - the cert is good .

Can sombeody share the config?
Thx a lot !

**Comment by binerf** [comment]
> Hi is the https://emby.media - emby revers proxy working for somebody ? I get Forbidden - the cert is good .
> 
> Can sombeody share the config?
> Thx a lot !

Hello,
Did you manage to make it works ?
I'm facing issue with "502 Bad Gateway" error....
Thx

**Comment by bikram990** [comment]
[Jenkins](https://www.jenkins.io/)

[OnlyOffice](https://www.onlyoffice.com/)

[PiHole](https://pi-hole.net/)

[Nexus](https://www.sonatype.com/products/repository-oss)

**Comment by FranticPanic** [comment]
Little late to the post, but didn't see this mentioned.

[Netdata](https://www.netdata.cloud/)

**Comment by AllofTheThangz** [comment]
Petio
Whoogle

**Comment by Vernoxvernax** [comment]
[Mango](https://github.com/hkalexling/Mango)
[Gotify](https://gotify.net/)
[wg-manager](https://github.com/perara/wg-manager)

**Comment by eitch** [comment]
[Proxmox UI](https://www.proxmox.com/en/proxmox-ve)
[ioBroker](https://smarthome.buanet.de/docker/container-images/iobroker/)
[Jarvis](https://github.com/Zefau/ioBroker.jarvis)
WebCam
[Node-Red](https://github.com/ioBroker/ioBroker.node-red)

**Comment by brandan-schmitz** [comment]
[Pterodactyl](https://pterodactyl.io/)
[Proxmox](https://www.proxmox.com/en/)
[Nextcloud](https://nextcloud.com/)
[Onlyoffice](https://www.onlyoffice.com/)

**Comment by PelleHanses** [comment]
Jitsi (https://jitsi.org/)
Nextcloud
Collabora core (Nextcloud office)
